src/services/item/plugins/app/chatBot/schemas.ts

```diff
@@ -32,6 +32,7 @@ export const create = {
     gptVersion: Type.Optional(
       Type.Enum(GPTVersion, { description: 'Model to use', default: OPENAI_GPT_VERSION }),
     ),
+    temperature: Type.Optional(Type.Number({ maximum: OPENAI_MAX_TEMP, minimum: OPENAI_MIN_TEMP })),
```

Comment:
Ah so you already have it. I guess this is sufficient.

Reply:
Yes, I found that after implementing the validation manually. I'll keep just that then.